.github/workflows/ci.yml

```diff
@@ -58,17 +64,73 @@ jobs:
           # Add the bin directory to PATH
           $OpenStudioBinPath = Join-Path $ExtractPath "bin"
           echo $OpenStudioBinPath >> $env:GITHUB_PATH
+
       - name: Download Weather
         run: uv run openstudio-hpxml-calibration download-weather
+
       - name: Measure unit tests
         run: |
           cd src/measures/ModifyXML/tests
           openstudio modify_xml_test.rb
+
       - name: Unit tests
         run: uv run pytest -v
+
       - name: Save test results
         uses: actions/upload-artifact@v4
         if: ${{ runner.os == 'Linux' && matrix.python-version == '3.13' }}
         with:
           name: test_results
           path: tests/results/*
+
+  calibrate:
```

Comment:
New CI job to run mini calibrations on all test files. Runs are split across 4 runners, so it only adds ~30 mins.